spec/intl.html

```diff
@@ -11,6 +11,70 @@ <h1>Amendments to the ECMAScript® 2021 Internationalization API Specification</
     </p>
   </emu-note>
 
+  <emu-clause id="sup-time-zone-names">
+    <h1><a href="https://tc39.es/ecma402/#sec-time-zone-names">Time Zone Names</a></h1>
+
+    <ins class="block">
+      <p>
+        <emu-xref href="#sec-time-zone-names"></emu-xref> defines a set of host-defined abstract operations concerning the names of supported time zones.
+        This section introduces additional requirements on how ECMAScript hosts that are conforming implementations of the ECMAScript 2021 Internationalization API Specification may implement these operations.
+      </p>
+    </ins>
+
+    <p>
+      The ECMAScript 2021 Internationalization API Specification identifies time zones using the Zone and Link names of the IANA Time Zone Database. Their canonical form is the corresponding Zone name in the casing used in the IANA Time Zone Database.
+    </p>
+
+    <p>
+      All registered Zone and Link names are allowed. <ins>Conforming</ins> implementations must recognize all such names, and use best available current and historical information about their offsets from UTC and their daylight saving time rules in calculations. However, the set of combinations of time zone name and language tag for which localized time zone names are available is implementation dependent.
+    </p>
+
+    <emu-clause id="sec-intl-hostisvalidtimezonename">
+      <h1><a href="https://tc39.es/ecma402/#sec-isvalidtimezonename"><ins>Host</ins>IsValidTimeZoneName</a> ( _timeZone_ )</h1>
+
+      <p>
+        <del>The</del> <ins>Host</ins>IsValidTimeZoneName <ins>is a host-defined</ins> abstract operation <ins>which</ins> verifies that the _timeZone_ argument (which must be a String value) represents a valid Zone or Link name of the IANA Time Zone Database.
```

Comment:
I believe that's the pre-existing requirement that Intl places on implementations — it must be based on a version of the IANA tz database, as recent as would be feasible.

Reply:
https://tc39.es/ecma402/#sec-time-zone-names:
>  The ECMAScript 2021 Internationalization API Specification identifies time zones using the Zone and Link names of the IANA Time Zone Database. Their canonical form is the corresponding Zone name in the casing used in the IANA Time Zone Database.
>
> All registered Zone and Link names are allowed. Implementations must recognize all such names, and use best available current and historical information about their offsets from UTC and their daylight saving time rules in calculations.

Reply:
This doesn't say other names are prohibited; I think the only part of the 402 spec prohibiting additional time zones would be the implementation of `CanonicalizeTimeZoneName` which doesn't allow for host-defined processing.

Does it makes sense to some way reconcile a host supporting a time zone that is not part of 402? If a time zone could be supported in a non-402 host, should it also be able to be present in a 402 host?

Reply:
This part of 402 is a bit unusual since it will be the only part of the 402 spec that places additional restrictions on what an engine may do, instead of allowing it extra capabilities. But I don't believe it's the intention of 402 that an engine may allow IANA time zone names plus whatever else.